src/go/languageservice/workspace.go

```diff
@@ -169,20 +171,54 @@ func (id WorkspaceID) GetWorkspaceErrors() []error {
 	return errors
 }
 
-func (id WorkspaceID) GetCompletions() ([]string, error) {
+func (id WorkspaceID) GetCompletions(position int) ([]string, error) {
 
 	workspace, err := id.getWorkspace()
 	if err != nil {
 		return nil, *err
 	}
 
 	completions := []string(nil)
-
-	// TODO: take into account context.
-	// TODO: support locals.
+    
+	// TODO: support other node types.
 	// TODO: return item type.
 	// TODO: support fetching item description.
 	for _, wd := range workspace.Files {
```

Comment:
Note that this should eventually look at just the current file as opposed to looping through all files... but I was lazy :)

Reply:
Oh, ok, I'll change it.

Reply:
No need to do this now, just a note.